src/modules/commander/Commander.cpp

```diff
@@ -1547,9 +1548,12 @@ Commander::run()
 			_was_falling = _land_detector.freefall;
 		}
 
+		if (_controller_status_sub.updated()) {
```

Comment:
You can move this within the auto disarm block below and only store a boolean (in the class) for launch_detection_running.

Reply:
Done in the other PR. I'll pull it in here. Did anything within the Fixed Wing land-detector need to be changed??